client/src/components/MagicalFactor/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import Keywords from '../../assets/Keywords';
+
+import './style.css';
+
+const MagicalFactor = ({ value: { text, src }, reviewText }) => (
+  <div className="magical">
+    <div className="magical__keyword">
+      <div className="magical__keyword__img">
+        <img src={Keywords[src]} alt={text} className="magical__keyword__img" />
```

Comment:
both the `img` and the wrapping `div` have the same class, that doesn't seem right. Should be enough to have the class on the div, if you need to add additional styles for the img, you can use `.magical__keyword__img img {}` selector, no need to add class for it (you can if you want to).
👍 for using BEM, however it should be `magical__keyword-img` (block__element--modifier; block: `magical` element: `keyword-img` and no modifier)

Reply:
done